runtime-linux.md

```diff
@@ -1,11 +1,5 @@
 # Linux Runtime
 
-## File descriptors
-
-By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
-The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
-Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
```

Comment:
> Do you also have problems with the move from runtime-linux.md to runtime.md? Or are you satisfied now that there are API-agnostic guarantees again?

I don't see why that question is relevant, but I'll answer it anyway: No I don't have a problem with moving stuff from `runtime-*.md` to `runtime.md` if it's applicable to all platforms. I do have a problem with moving stuff from `runtime-*.md` to `cli-interface-*.md` because it makes it appear not applicable to the entire runtime. Does my logic behind that point make sense to you now?